pkg/pool-stable/test/foundry/LiquidityApproximationStable.t.sol

```diff
@@ -160,11 +161,21 @@ contract LiquidityApproximationStableTest is LiquidityApproximationTest, StableP
     function _setAmplificationParameter(address pool, uint256 newAmplificationParameter) private {
         uint256 updateInterval = 5000 days;
 
+        (uint256 value, bool isUpdating, ) = IStablePool(pool).getAmplificationParameter();
+
         vm.prank(admin);
-        StablePool(pool).startAmplificationParameterUpdate(newAmplificationParameter, block.timestamp + updateInterval);
+        uint256 endTime = block.timestamp + updateInterval;
+        StablePool(pool).startAmplificationParameterUpdate(newAmplificationParameter, endTime);
+
+        (AmplificationState memory ampState, uint256 precision) = IStablePool(pool).getAmplificationState();
+        assertEq(ampState.startTime, block.timestamp, "Wrong amp update start time");
+        assertEq(ampState.endTime, endTime, "Wrong amp update end time");
+        assertEq(ampState.startValue, value, "Wrong amp update start value");
+        assertEq(ampState.endValue, newAmplificationParameter * precision, "Wrong amp update end value");
+
         vm.warp(block.timestamp + updateInterval + 1);
 
-        (uint256 value, bool isUpdating, uint256 precision) = StablePool(pool).getAmplificationParameter();
+        (value, isUpdating, ) = IStablePool(pool).getAmplificationParameter();
```

Comment:
this is a kinda weird place for this test :sweat_smile: 
I'd test the getter elsewhere, preferably in `StablePool.t.sol`

Reply:
It was just convenient, since this is where we're testing the amp update, and the main stable pool test is pretty empty. I can move it.